src/uct/api/uct.h

```diff
@@ -562,6 +581,18 @@ struct uct_iface_attr {
             } rndv;                      /**< Attributes related to rendezvous protocol */
         } tag;                           /**< Attributes for TAG operations */
 
+        struct {
```

Comment:
i would suggest the following:
```
struct {
    uint64_t op_mask;
    uint64_t fop_mask;
} atomic32, atomic64;
```

Reply:
fixed

Reply:
why it is called MASK ? why not just op and fop

Reply:
because it is bitmask to let to know about supported operations

Reply:
cap ? for capabilities ?

Reply:
you know what, what naming for such capabilities we use in another datastructures ?

Reply:
it's already contained in a structure with the name "cap", maybe use op_flags and fop_flags to be same naming as the general "flags"

Reply:
or just ```ops``` and ```fops``` to simplify reading

Reply:
@yosefe renamed & squashed